src/components/Contexts/StepperDialogProvider.jsx

```diff
@@ -1,30 +1,43 @@
-import React, { createContext, useCallback, useState } from 'react'
+import React, { createContext, useCallback, useEffect, useState } from 'react'
 
 const StepperDialogContext = createContext()
 
 const StepperDialogProvider = ({ children }) => {
   const [isStepperDialogOpen, setIsStepperDialogOpen] = useState(false)
   const [stepperDialogTitle, setStepperDialogTitle] = useState('')
+  const [allCurrentPagesDefinitions, setAllCurrentPagesDefinitions] = useState(
+    []
+  )
   const [allCurrentPages, setAllCurrentPages] = useState([])
-  const [currentPage, setCurrentPage] = useState(1)
+  const [currentPageIndex, setCurrentPageIndex] = useState(1)
+
+  useEffect(() => {
+    if (allCurrentPagesDefinitions.length > 0) {
+      const allCurrentPagesDefinitionsSorted = allCurrentPagesDefinitions.sort(
```

Comment:
Is the const really needed here since we only use it once?

Reply:
Je comprends, d'autant qu'il n'y a qu'un `sort` d'effectué ici, mais je trouve malgré tout plus propre et lisible de procéder ainsi, pour le faible coût que ça peut avoir ?

Reply:
No issue for me it is not a blocking comment (like all my comments on this PR) just an observation/preference